### ISSUE TITLE
Roadmap considerations of Afforms?

### ISSUE THREAD

**Issue by ginkgomzd** [progressivetech/net.ourpowerbase.remoteform#21]
Has there been any consideration yet of the impact of the new Civi FormBuilder (Afforms) and whether it is desirable or feasible to integrate with RemoteForm?

I asked a similar question here: https://lab.civicrm.org/extensions/afform/-/issues/19

**Comment by jmcclelland** [comment]
Yes - I'm 100% in favor of trying to get this functionality included in afform or rewriting this extension to work with afform (see for example https://github.com/progressivetech/net.ourpowerbase.remoteform/issues/7#issuecomment-445284318). This extension was written after afform was started, but from the beginning has been considered at best a shim until afform is complete and at worst an experiment to help inform afform development.
